configurations/admin.template.ini

```diff
@@ -1037,6 +1037,14 @@ moduls.limitAllowedActions.label = "Limit allowed actions"
 moduls.limitAllowedActions.permissionName = FEATURE_LIMIT_ALLOWED_ACTIONS
 moduls.limitAllowedActions.group = GROUP_ENABLE_DISABLE_FEATURES
 
+moduls.feedWithLiveEntries.enabled = true
+moduls.feedWithLiveEntries.permissionType = 2
+moduls.feedWithLiveEntries.label = "Show live entries in feed"
```

Comment:
@TalDubovKaltura  semantics again, but I'd replace the word "Show" with "Include"

Reply:
Done.